core/sessions/orm.go

```diff
@@ -152,7 +196,10 @@ func (o *orm) CreateSession(sr SessionRequest) (string, error) {
 	if len(uwas) == 0 {
 		lggr.Infof("No MFA for user. Creating Session")
 		session := NewSession()
-		_, err = o.db.Exec("INSERT INTO sessions (id, last_used, created_at) VALUES ($1, now(), now())", session.ID)
+		err := o.q.Transaction(func(tx pg.Queryer) error {
+			_, err = o.q.Exec("INSERT INTO sessions (id, email, last_used, created_at) VALUES ($1, $2, now(), now())", session.ID, user.Email)
```

Comment:
This looks like it might be able to race to me. What if a request in parallel is also creating a session?

Reply:
Removed transaction for now as it was previously